src/cryptoadvance/specter/templates/wallet/send/new/components/fee_selection.jinja

```diff
@@ -1,18 +1,28 @@
 <div class="fee_container">
     <div>
-        <label><input type="checkbox" class="inline" name="subtract" value="1"> Subtract fees from amount</label>
+        <label><input type="checkbox" class="inline" name="subtract" id="subtract" value="1" onchange="toggleSubtractFrom(this)"> Subtract fees from amount</label>
+        <div class="tool-tip" style="text-align: center;">
+            <i class="tool-tip__icon">i</i>
+            <p class="tool-tip__info">
+                <span class="info">
+                    <span class="info__title">Subtract fees from amount<br><br></span>
+                    If checked, the transaction fees will be payed off from the tranaction amount.<br><br>Otherwise, the fees will be payed as an added cost in addition to the amount sent.
```

Comment:
```suggestion
                    If checked, the transaction fees will be payed off from the transaction amount.<br><br>Otherwise, the fees will be payed as an added cost in addition to the amount sent.
```
Could these lines be divided too? They are very long.